docs/api/instruments/philips/index.rst

```diff
@@ -0,0 +1,12 @@
+.. module:: pymeasure.instruments.philips
+
+#######
+Philips
+#######
+
+This section contains specific documentation on the Philips instruments that are implemented. If you are interested in an instrument not included, please consider :doc:`adding the instrument </dev/adding_instruments>`.
+
+.. toctree::
+   :maxdepth: 2
+
+   PM6669
```

Comment:
When you generate the docs using ` make html` in the docs directory, does that page come out correctly/usefully? 
Your PM6669.rst will need some skeleton content like other instruments to, e.g. https://raw.githubusercontent.com/pymeasure/pymeasure/master/docs/api/instruments/parker/parkerGV6.rst